lang/java/reef-tang/tang/src/test/java/org/apache/reef/tang/TestClassLoaders.java

```diff
@@ -20,36 +20,40 @@
 
 import org.apache.reef.tang.exceptions.BindException;
 import org.apache.reef.tang.exceptions.InjectionException;
-import org.apache.reef.tang.exceptions.NameResolutionException;
 import org.apache.reef.tang.types.ClassNode;
 import org.junit.Assert;
+import org.junit.Test;
 
 import java.io.File;
+import java.io.FilenameFilter;
 import java.net.MalformedURLException;
 
 public class TestClassLoaders {
 
-  //  @Test
-  public void testOneJar() throws MalformedURLException,
-      ClassNotFoundException, NameResolutionException, BindException {
+  private static final FilenameFilter JAR_FILE_FILTER = new FilenameFilter() {
+    public boolean accept(final File dir, final String name) {
+      return name.endsWith(".jar");
```

Comment:
Ideally, we would have a single place in which we write release version, and all other places in code/config which require it consume it from that place. In practice, we're not there yet.

As a compromise, we can create a constant in this file, so that the version is hardcoded once, and then use it to construct exact file names.